src/main/java/uk/gov/digital/ho/hocs/queue/ukvi/UKVIComplaintData.java

```diff
@@ -42,22 +47,31 @@ public String getComplaintType() {
     }
 
     @Override
-    public ComplaintCorrespondent getComplaintCorrespondent() {
-        String applicantType = ctx.read(APPLICANT_TYPE);
-        ComplaintCorrespondent correspondent;
+    public List<ComplaintCorrespondent> getComplaintCorrespondent() {
 
-        if (applicantType.equals("APPLICANT")) {
-            correspondent = new ComplaintCorrespondent(ctx.read(APPLICANT_APPLICANT_NAME));
-            optionalString(ctx, APPLICANT_APPLICANT_EMAIL).ifPresent(correspondent::setEmail);
-            optionalString(ctx, APPLICANT_APPLICANT_PHONE).ifPresent(correspondent::setTelephone);
+        List<ComplaintCorrespondent> correspondents = new ArrayList<>();
 
-        } else if (applicantType.equals("AGENT")) {
-            correspondent = new ComplaintCorrespondent(ctx.read(AGENT_APPLICANT_NAME));
-
-        } else {
-            throw new IllegalStateException("APPLICANT_TYPE Unknown : " + applicantType);
+        try {
+            String applicantType = ctx.read(APPLICANT_TYPE);
+            if (applicantType.equals("APPLICANT")) {
+                ComplaintCorrespondent applicantCorrespondent = new ComplaintCorrespondent(ctx.read(APPLICANT_APPLICANT_NAME), CorrespondentType.COMPLAINANT);
+                optionalString(ctx, APPLICANT_APPLICANT_EMAIL).ifPresent(applicantCorrespondent::setEmail);
+                optionalString(ctx, APPLICANT_APPLICANT_PHONE).ifPresent(applicantCorrespondent::setTelephone);
+                correspondents.add(applicantCorrespondent);
```

Comment:
Maybe a comment to say that the first entry in the List will be the primary correspondent.